decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/util/HTTPSFactory.java

```diff
@@ -90,19 +84,7 @@ public static TrustManagerFactory createTrustManager(String truststorePath, Stri
 
         TrustManagerFactory tmf = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
 
-        // initialize certification path checking for the offered certificates and
-        // revocation checks against CLRs
-        CertPathBuilder cpb = CertPathBuilder.getInstance("PKIX");
-        PKIXRevocationChecker rc = (PKIXRevocationChecker) cpb.getRevocationChecker();
-        rc.setOptions(EnumSet.of(PKIXRevocationChecker.Option.PREFER_CRLS, // prefer CLR over OCSP
-                PKIXRevocationChecker.Option.SOFT_FAIL, PKIXRevocationChecker.Option.NO_FALLBACK));
-        // TODO: Remove SOFT_FAIL and change to ONLY_END_ENTITY
-        // and don't fall back to OCSP checking.
-
-        PKIXBuilderParameters pkixParams = new PKIXBuilderParameters(keystore, new X509CertSelector());
-        pkixParams.addCertPathChecker(rc);
-
-        tmf.init(new CertPathTrustManagerParameters(pkixParams));
+        tmf.init(keystore);
```

Comment:
Why was this removed?
Since the tests pass I would assume this is fine, but I would like to know some more context to this change.

Reply:
Because there is no need to check for revoked certificates if all financial institutions are root CAs that cannot revoke themselves.